src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -333,8 +333,11 @@ export class TabsTitleControl extends TitleControl {
 				// color, the two colors would collide and the tab border never shows up.
 				// see https://github.com/Microsoft/vscode/issues/33111
 				const activeTabBorderColor = this.getColor(isGroupActive ? TAB_ACTIVE_BORDER : TAB_UNFOCUSED_ACTIVE_BORDER);
+				const activeTabBorderColorTop = this.getColor(isGroupActive ? TAB_ACTIVE_BORDER_TOP : TAB_UNFOCUSED_ACTIVE_BORDER);
 				if (activeTabBorderColor) {
 					tabContainer.style.boxShadow = `${activeTabBorderColor} 0 -1px inset`;
+				} else if (activeTabBorderColorTop) {
+					tabContainer.style.boxShadow = `${activeTabBorderColorTop} 0 2px inset`;
```

Comment:
I would suggest to use `1px` here to keep a similar look as the bottom border.

Reply:
Done. Thanks for the feedback.